services/tests/literpc_tpu_quic_server_integrationtest.rs

```diff
@@ -0,0 +1,478 @@
+use anyhow::bail;
+use countmap::CountMap;
+use crossbeam_channel::{Receiver, RecvError, RecvTimeoutError, Sender};
+use futures::future::join_all;
+use log::{debug, error, info, trace, warn};
+use quinn::TokioRuntime;
+use solana_lite_rpc_core::quic_connection_utils::QuicConnectionParameters;
+use solana_lite_rpc_core::structures::identity_stakes::IdentityStakes;
+use solana_lite_rpc_core::tx_store::empty_tx_store;
+use solana_lite_rpc_services::tpu_utils::tpu_connection_manager::TpuConnectionManager;
+use solana_rpc_client::rpc_client::SerializableTransaction;
+use solana_sdk::hash::Hash;
+use solana_sdk::instruction::Instruction;
+use solana_sdk::message::Message;
+use solana_sdk::pubkey::Pubkey;
+use solana_sdk::signature::{Keypair, Signature, Signer};
+use solana_sdk::signer::keypair;
+use solana_sdk::transaction::{Transaction, VersionedTransaction};
+use solana_streamer::nonblocking::quic::ConnectionPeerType;
+use solana_streamer::packet::PacketBatch;
+use solana_streamer::quic::StreamStats;
+use solana_streamer::streamer::StakedNodes;
+use solana_streamer::tls_certificates::new_self_signed_tls_certificate;
+use std::collections::HashMap;
+use std::net::{IpAddr, Ipv4Addr, SocketAddr, UdpSocket};
+use std::ops::Deref;
+use std::option::Option;
+use std::path::Path;
+use std::str::FromStr;
+use std::sync::atomic::{AtomicBool, AtomicU32, Ordering};
+use std::sync::{Arc, RwLock};
+use std::thread;
+use std::time::{Duration, Instant};
+use tokio::runtime::{Builder, Runtime};
+use tokio::sync::broadcast;
+use tokio::sync::broadcast::error::SendError;
+use tokio::task::JoinHandle;
+use tokio::time::{interval, sleep};
+use tracing_subscriber::util::SubscriberInitExt;
+use tracing_subscriber::{filter::LevelFilter, fmt};
+
+#[derive(Copy, Clone, Debug)]
+struct TestCaseParams {
+    sample_tx_count: u32,
+    stake_connection: bool,
+}
+
+const MAXIMUM_TRANSACTIONS_IN_QUEUE: usize = 200_000;
+const MAX_QUIC_CONNECTIONS_PER_PEER: usize = 8; // like solana repo
+
+const QUIC_CONNECTION_PARAMS: QuicConnectionParameters = QuicConnectionParameters {
```

Comment:
Should be exactly like lite-rpc, could you make const variable for both of them, usually most of them are already constant.